lib/@types/Application.ts

```diff
@@ -0,0 +1,19 @@
+export type Application = {
+  id: string;
+  name: string;
+  description: string;
+  defaultHomePage: string;
+  redirectUris: string[];
+};
+
+export type ApplicationInput = {
+  id: string;
+  description: string;
+  name: string;
+  redirectUris: string[];
+  defaultHomePage: string;
+};
```

Comment:
I understand why you did this, but we probably can ride on one of the two :). Not sure what to think though.